### ISSUE TITLE
Message body lost when message copied to sent folder

### ISSUE THREAD

**Issue by dfelicia** [UniversalMailer/UniversalMailer#83]
Mail server is Oracle Beehive. This started happening only after upgrading to Sierra. Can reproduce using latest UniversalMailer 3.0 build 15.

Steps:

Enable plugin, and set preferences to use Helvetica-11, points instead of pixels
Compose and send a rich text mail
Check sent folder in Apple Mail. It looks OK.
Check sent folder in any other mail client (including telnet). The message body is gone.

Disabling the plugin eliminates the issue.

I'm sure there is some nuance with Sierra and Oracle Beehive, here... nonetheless, it works fine until the plugin is enabled.

**Comment by UniversalMailer** [comment]
I might be, but it would be good to know what's different... Can you try sending me a couple of messages at universal.mailer.noware at gmail.com with the plugin enabled? Also: can you tell if something similar happens with other email accounts (e.g. Gmail or others)?

**Comment by dfelicia** [comment]
Yes, I’ll do that. One thing I just noticed, though, and this definitely seems like a bug in UM…

Mail > Preferences > Composing. Format is set to Plain Text.
Mail > Preferences > Fonts and Colors. Message Font: Calibri 15, Fixed-width font: Menlo Regular 12
Mail > Preferences > UniversalMailer. Calibri-12, points instead of pixels

Compose a new message. Go to Format > Make Rich Text. Start typing some text. Try to make bold or italics, and observe it’s not possible. Try to change the font. Not possible. Exit, disable UM (removed from Bundles dir), start Mail, run same test. Now Bold, italics etc.s all work.

So UM is interfering (in a bad way) with composer. This is new in Sierra - I’ve been using UM, happily, for years.

Let me turn a few more knobs and send the requested test messages. It’s possible that only replies are affected (I really only send rich text when it’s a reply to a message that was sent to me in rich text).

**Comment by dfelicia** [comment]
I've confirmed with a colleague that my issue can be reproduced. So it does appear that UM 3b15 does have some kind of issue with Mail 10.x on Sierra, but it also seems like it only affects some kinds of e-mail accounts/servers (Oracle Beehive, for certain).

**Comment by dfelicia** [comment]
Note: Upgraded to 10.12.1 and the issue persists.

**Comment by hub601** [comment]
I experienced the same issue with build 16 and had to disable UM. Just tried again with build 19 and the issue is still there. Is there any hope for a fix?
